plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/storage/LibvirtStorageAdaptor.java

```diff
@@ -692,6 +702,10 @@ public KVMStoragePool createStoragePool(String name, String host, int port, Stri
                 sp.create(0);
             }
 
+            if (type == StoragePoolType.NetworkFilesystem) {
+                checkNetfsStoragePoolMounted(name);
```

Comment:
@rp- can check mounted or not after _createNetfsStoragePool()_ call above?

Reply:
as far as I understood, the `createNetfsStoragePool()` only defines the libvirt XML storage pool
and it will activated/created/mounted in the `sp.create(0)` call

Reply:
yes, got it.